examples/devices/lighty-network-topology-device/src/test/java/io/lighty/netconf/device/topology/DeviceTest.java

```diff
@@ -112,6 +118,44 @@ public void getSchemaTest() throws IOException, URISyntaxException, SAXException
         }
     }
 
+    @Test
+    public void topologyRpcsTest() throws ExecutionException, InterruptedException, IOException, URISyntaxException,
+            SAXException, TimeoutException {
+        final SimpleNetconfClientSessionListener sessionListener = new SimpleNetconfClientSessionListener();
+        try (NetconfClientSession session = dispatcher.createClient(createSHHConfig(sessionListener)).get()) {
+            final Document expectedValuesDoc = XmlUtil.readXmlToDocument(xmlFileToInputStream(EXPECTED_VALUES_XML));
+
+            final NetconfMessage createTopoResponse = performNetconfRequest(CREATE_TOPOLOGY_REQUEST_XML,
+                    sessionListener);
+            validateNetconfResponse(createTopoResponse, expectedValuesDoc, "create_topo");
```

Comment:
refactor to assert specific nodes (this is not validating)
do not use unified multi-purposed methods -> we need to see here something like: assert that createTopoResponse has element ok and message-id m-1

same for other assertions